feeder/feeder.py

```diff
@@ -343,6 +380,20 @@ def fetch_exchanges():
             statsd.increment('fetch_one_exchange_error')
             logger.exception('Exception while fetching exchange of %s' % k)
 
+def fetch_prices():
```

Comment:
I'd recommend naming this function (and other related functions) `fetch_price`. There's no reason to have it in plural.